_posts/2022-07-08-dev-sprint-leaders.md

```diff
@@ -0,0 +1,55 @@
+---
+author: DjangoCon US Organizers
+category: General
+date: 2022-07-08 13:00:00
+layout: post
+title: "Lead or organize a Development Sprint at DCUS 2022!"
+---
+
+The last two days of DjangoCon US *(this year, October 20-21)* are devoted to Development Sprints. This blog post is targeted at current open source maintainers and contributors who are considering organizing or leading a Development Sprint at DCUS 2022. As such, I won't be defining "Development Sprint". That's for another blog post.
+
+## Why lead/organize a sprint at DCUS 2022?
+There are as many reasons to lead or organize a development sprint as there are projects to work on and potential organizers. Some of the key reasons to consider are:
+- Meet & work with current contributors in person!
+- Test & refine your new user & new contributor docs with live feedback. Better new contributor docs lead to better new issues, bug fixes and PRs for your project.
+- Build/improve a workflow for virtual attendees that your project can continue to use after DjangoCon US (when EVERYONE will be 'virtual').
```

Comment:
> Build/improve

Change to "Build (or improve)" to get better parallelism with the first two bullets

Reply:
Oh! I like improved parallelism!